servant-server/servant-server.cabal

```diff
@@ -65,7 +67,8 @@ library
       , wai                >= 3.0  && < 3.1
       , wai-app-static     >= 3.0  && < 3.2
       , warp               >= 3.0  && < 3.2
-
+      , word8              >= 0.1.0
```

Comment:
We've tried to stick to having both upper and lower bounds, and updating as new versions get released.